db/seeds.d/107-enable_dynflow.rb

```diff
@@ -4,5 +4,5 @@
 # !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
 #
 User.as(::User.anonymous_api_admin.login) do
-  Setting.find_by(:name => "dynflow_enable_console").update!(:value => true) if Rails.env.development?
+  Setting.find_by(:name => "dynflow_enable_console")&.update!(:value => true) if Rails.env.development?
```

Comment:
```suggestion
    Setting["dynflow_enable_console"] = true if Rails.env.development? && !Setting["dynflow_enable_console"]
```

Reply:
`dynflow_enable_console` is not a Katello setting. Why should we add it here if its owner can take care of it?

Reply:
When foreman-tasks is loaded, I expect `dynflow_enable_console` setting is set to `true` no matter if it's product or development environment. So this file may be out of date already. 
Could we try remove this file and set up the devel environment then check if it works?

Reply:
Sounds good to me. @chris1984 do you mind trying this?